app/backfill-synonym-slugs.js

```diff
@@ -0,0 +1,67 @@
+import { DynamoDBClient, paginateScan } from '@aws-sdk/client-dynamodb'
+import { GetParameterCommand, SSMClient } from '@aws-sdk/client-ssm'
+import { BatchWriteCommand } from '@aws-sdk/lib-dynamodb'
+import { unmarshall } from '@aws-sdk/util-dynamodb'
+import { slug } from 'github-slugger'
+import chunk from 'lodash/chunk'
+
+async function getTableNameFromSSM(dynamoTableName) {
+  const ssmClient = new SSMClient({ region: 'us-east-1' })
+
+  try {
+    const command = new GetParameterCommand({ Name: dynamoTableName })
+    const response = await ssmClient.send(command)
+
+    if (!response.Parameter?.Value) {
+      throw new Error('dynamoTableName not found in SSM')
+    }
+
+    return response.Parameter.Value
+  } catch (error) {
+    console.error('Error fetching table name from SSM:', error)
+    throw error
+  }
```

Comment:
FWIW, you could safely remove this entire `try`/`except` block. You'll get a good stack trace either way.